docs/tsg_install.md

```diff
@@ -33,6 +37,20 @@ pvc-febcc15e-25d7-11e8-92c2-0a58ac1f1190-rep-578b5bcc6b-zkhn8   1/1       Runnin
 
 The AKS cluster runs ubuntu 16.04 LTS with the kubelet running in a container (debian-jessie 8). The kubelet logs show the absence of the iSCSI initiator. Hence, the volume is not attached to the node. Configuring kubelet to run with iSCSI utils should fix this issue. For more information, see  https://github.com/openebs/openebs/issues/1335.
 
+## Issue: On Azure, OpenEBS installation is failing through helm due to RBAC
+
+## <a name="RBACNotEnableAzure"></a>
+
+### Troubleshooting the issue and Workaround:
+
+On AKS, while installation of OpenEBS through helm, if you get following error.
```

Comment:
On AKS, while installing OpenEBS using Helm, if you get the following error, you must enable RBAC on Azure. For more details, see [prerequisites](https://staging-docs.openebs.io/docs/next/prerequisites.html).